src/serde_macros.rs

```diff
@@ -1,86 +1,124 @@
+#[macro_use]
```

Comment:
So this is probably no longer needed once you move the macro out of this submodule.